### ISSUE TITLE
Unable to use 2-fingers-up gesture on some screens to read full page from top when VoiceOver on 

### ISSUE THREAD

**Issue by qanitah** [cds-snc/covid-shield-mobile#215]
While VoiceOver is on and 2 fingers up gesture is used to read the page from top, it doesn't work for most screens. When 2 fingers are flicked up on a screen, it should read all content on the page, starting from top.

It seems to work on screens that don't have the "COVID Alert" image on top of the page. 

Please see screenshots of one screen where it works and one where it doesn't.
iOS 13.5.1 on iPhone 7

![covidalert5](https://user-images.githubusercontent.com/14280062/85895622-08111900-b7c5-11ea-8798-42ac0e305edd.jpg)
![covidalert4](https://user-images.githubusercontent.com/14280062/85895623-08a9af80-b7c5-11ea-9f75-83a6f7889dd8.jpg)

**Comment by timarney** [comment]
@smcmurtry let's chat about this one - I know at least what's happening here now.

**Comment by smcmurtry** [comment]
This is because we are using the header as a button to open the demo menu: 
`<TouchableWithoutFeedback onPress={onLogoPress}>`.

I tried removing this and voiceover works as expected. So this will not be an issue when we remove this feature pre-release.

**Comment by JuliannaR** [comment]
@smcmurtry sounds good. @timarney and I talked about this, this morning. That works for all screens with the header. There are still the screens with the close/cancel in the top right that experience a similar pattern problem, Are we able to address those?

**Comment by smcmurtry** [comment]
This looks like it is now fixed: https://youtu.be/qtn_GOXqXQ4. Not sure when that happened.

**Comment by JuliannaR** [comment]
@smcmurtry gestures now work throughout the app on IOS. :)
